packages/create-dapp/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@agoric/create-dapp",
+  "version": "0.1.0",
+  "description": "Create an Agoric Javascript smart contract application",
+  "type": "module",
+  "bin": {
+    "create-dapp": "src/create-dapp.js"
+  },
+  "files": [
+    "src"
+  ],
+  "scripts": {
+    "build": "exit 0",
+    "test": "ava",
+    "test:c8": "c8 $C8_OPTIONS ava --config=ava-nesm.config.js",
+    "test:xs": "exit 0",
+    "lint-fix": "yarn lint:eslint --fix",
+    "lint": "run-s --continue-on-error lint:*",
+    "lint:types": "tsc -p jsconfig.json",
+    "lint:eslint": "eslint ."
+  },
+  "devDependencies": {
+    "ava": "^5.2.0",
+    "c8": "^7.13.0",
+    "dd-trace": "^3.3.0"
```

Comment:
I think we'll need to match these to currently used versions of deps in other packages.

Reply:
Updated.

Actually on second though, do we need `dd-trace` package here?

Reply:
I believe we don't. Good call.